package.json

```diff
@@ -51,6 +52,8 @@
     "mongoose": "^4.9.6",
     "msw": "^0.25.0",
     "multer": "^1.4.2",
+    "nano": "^9.0.3",
+    "node-couchdb": "^1.3.0",
```

Comment:
are you using this `node-couchdb` package?

Reply:
I was at first but then switched to nano, so I should uninstall it.